index.bs

```diff
@@ -7,3 +7,514 @@ URL: https://netwerk-digitaal-erfgoed.github.io/requirements-network-of-terms/
+such as diverging data formats or metadata schemas.
+
+The Network of Terms will initially use [GraphQL](https://graphql.org/) as its interface, \[JSON](https://en.wikipedia.org/wiki/JSON) as its data format
+and \[SKOS](https://en.wikipedia.org/wiki/Simple_Knowledge_Organization_System) as its metadata schema.
+GraphQL, JSON and SKOS are relatively simple and known to software developers, which fosters easy adoption.
+Other interfaces, data formats or metadata schemas may be added later.
+
+<div class="note">
+    **Discussion**: GraphQL is a query language, but it doesn't support search operators, such as wildcards.
+    Which standard or best practice should the Network of Terms use for this?
+</div>
+
+<div class="note">
+    **Discussion**: is SKOS sufficient to express all information about terms?
+    For example, terminology source A can use metadata schema X to indicate the "status" of a term (such as a candidate term),
+    while terminology source B can use metadata schema Y for this.
+    SKOS has no support to describe the "status". Should a different metadata schema be used for this — or an own ontology?
+</div>
+
+A uniform endpoint means that the Network of Terms offers the same search capabilities for all terminology sources.
+This, however, only applies to sources that meet
+the [Requirements for terminology sources](https://netwerk-digitaal-erfgoed.github.io/requirements-terminologiebronnen/) (in Dutch).
+If a source doesn't support certain functionality, the Network of Terms doesn't support it either.
+If a source has more functionality than stipulated by the requirements, the Network of Terms cannot use it.
+
+The Network of Terms makes terms findable via an online search environment that is accessible to users {#2}
+--------------------
+**Priority**: Should have
+
+The Network of Terms has an endpoint for querying terminology sources (see [requirement 1](#1)).
+However, an endpoint is meant to be used by the systems of users, not directly by users.
+Yet users want to be able to query the Network of Terms to get an impression of the possibilities and the results,
+regardless of how the users' systems use it.
+The Network of Terms therefore has an online search environment for users.
+An example of such an environment is the [demonstrator](https://termennetwerk.netwerkdigitaalerfgoed.nl/) of the Network of Terms.
+
+The Network of Terms searches terminology sources registered in the Registry {#3}
+--------------------
+**Priority**: Must have
+
+The Network of Terms searches terminology sources, but does not determine which sources can be used.
+For this purpose the Network of Terms relies on the Registry. The Registry is the generic facility of the Dutch Digital Heritage Network
+that keeps track of the datasets in the network, including the available terminology sources.
+Use of the Registry improves clarity: anyone — both man and machine — who wants to do anything with datasets can use the Registry as an entry point.
```

Comment:
Let’s change this to ‘both human(s) and machine(s)’ to make it very clear women are included, too.

Reply:
👍🏻